sw/device/tests/BUILD

```diff
@@ -4101,6 +4101,37 @@ _RV_DM_JTAG_LC_STATES = get_lc_items(
     CONST.LCV.RMA,
 )
 
+[
+    opentitan_test(
+        name = "rv_dm_csr_rw_{}".format(lc_state),
+        srcs = ["example_test_from_flash.c"],
+        cw310 = hyper310_jtag_params(
+            otp = _SIVAL_OTP_IMAGE[lc_state],
+            tags = [
+                "lc_{}".format(lc_state),
```

Comment:
is this tag used? I don't think we consistently tag lc states

Reply:
I was following flash_ctrl_rma_test which I think was the example sival test in Miguel's PR. But apparently nobody else has added the tag to their tests. Do you want me to remove the tag?

Reply:
I think the tags are just used to inconsistently to be useful I guess it does not hurt to have them